deployment/ansible/roles/model-my-watershed.app/templates/nginx-app.conf.j2

```diff
@@ -1,3 +1,10 @@
+# Protects against scenarios where DNS records for the
+# site being proxied change.
+resolver 8.8.8.8 valid=300s;
```

Comment:
I had to hit the page twice to get cached results. That might have been a fluke though. It looks good.